optional_plugins/runner_remote/tests/test_remote.py

```diff
@@ -104,8 +102,7 @@ def test_run_suite(self):
         cmd_line = ('avocado run --force-job-id '
                     '0000000000000000000000000000000000000000 --json - '
                     '--archive /tests/sleeptest.py /tests/other/test '
-                    'passtest.py --mux-yaml ~/avocado/tests/foo.yaml '
-                    '~/avocado/tests/bar/baz.yaml --dry-run --filter-'
+                    'passtest.py --dry-run --filter-'
```

Comment:
I agree that there should be no dependency, but there's a clear deviation in the runner remote plugin, where some options are whitelisted and explicit support exists.  For these look at:

https://github.com/avocado-framework/avocado/blob/master/optional_plugins/runner_remote/avocado_runner_remote/__init__.py#L445

So we can either go ahead and drop that future alongside these test changes, but there's some regression in functionality and a improvement in code quality and maintenance.

Reply:
Hmmm, I see.

I vote for dropping the feature, but not sure about all the implications. IMO, makes no sense to a plugin depend on another optional plugin and assumes that it will be installed.

I'm thinking about alternatives...

Reply:
@clebergnu since that the remote plugin will be deprecated anyway, can you accept this?

Reply:
Yes, in light of those changes, I see no harm.